### PR TITLE
Updated docs for strides and dilations attributes 

### PR DESCRIPTION
**Pull request by JamesAllingham**
The docs for the `strides` and `dilations` attributes of various convolutional and pooling layers did not mention the defaults. I copied the way the defaults were described for the `pads` attribute.

### CODE REVIEW

File: onnx/defs/nn/defs.cc [hunk 1]
```diff
@@ -876,17 +882,24 @@ computes the output.)DOC";
...
         std::string("NOTSET"));
-    schema.Attr("pads", pads_doc, AttributeProto::INTS, OPTIONAL);
+    schema.Attr(
+        "pads",
+        pads_doc,
+        AttributeProto::INTS,
+        OPTIONAL);
     schema.Attr(
         "group",
         "number of groups input channels and output channels are divided into.",
```
Comment (wschin) [inline onnx/defs/nn/defs.cc]: What's the default value of `group`?

**Comment by wschin** [comment]
Do you mean that you make some `undefined` behaviors to `defined`? If yes, please bump their version numbers.

**Comment by JamesAllingham** [comment]
> 
> 
> Do you mean that you make some `undefined` behaviors to `defined`? If yes, please bump their version numbers.

So the docs didn't mention the

File: onnx/defs/nn/defs.cc (hunk 1 above)
Comment (JamesAllingham) [reply onnx/defs/nn/defs.cc]: Good catch! The default value is 1. I'll add that.

**Comment by JamesAllingham** [comment]
> 
> 
> Do you mean that you make some `undefined` behaviors to `defined`? If yes, please bump their version numbers.

Yes, I suppose I did... these defaults were mentioned in [conv.py](https://github.com/onnx/onnx/blob/57372f39631fddc720287a4703040c0551d4ee48/onnx/backend/test/case/node/conv.py) but I suppose that isn't a proper way to define things. I'll bump the version numbers.

**Comment by wschin** [comment]
Could you merge the conflicts? I think we can just keep both of your and master's changes and rerun document/test update.

**Pull request merged**